app/Models/TestModel.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class TestModel extends Model
+{
+    protected $table = 'tests';
+    protected $primaryKey = 'id';
```

Comment:
Not needed. It is automatically implied by Laravel, as already used in most other models.